internal/cluster/transition.go

```diff
@@ -482,6 +483,21 @@ func (th *transitionHandler) PostRefreshCluster(reason string) stateswitch.PostT
 				reason, params.eventHandler, extra...)
 		}
 
+		//report cluster install duration metrics in case of error. Cancel and Installed cases are
+		//treated separately in CancelInstallation and CompleteInstallation respectively
+		if sCluster.srcState != swag.StringValue(sCluster.cluster.Status) {
+			if reason == statusInfoError {
+				params.metricApi.ClusterInstallationFinished(params.ctx, "error", sCluster.srcState,
```

Comment:
please use models.ClusterStatusError instead of "error"

Reply:
It's looks the same but it's not. It's just the label of the dashboard. I did not want to put those label in constants so they will be used for other places by mistake. And I can not use the states either because it will create discrepancies with previous dashboards (for the other states other then error)

Reply:
FIXED